modules/tinymce/src/core/main/ts/api/Formatter.ts

```diff
@@ -133,7 +133,16 @@ const Formatter = function (editor: Editor): Formatter {
      * @param {Node} node Optional node to check.
      * @return {boolean} true/false if the specified selection/node matches the format.
      */
-    match: Fun.curry(MatchFormat.match, editor),
+    match: (name, vars?, node?) => Rtc.matchFormat(editor, name, vars, node),
+
+    /**
+     * Finds the closest matching format from a set of formats at the current selection.
```

Comment:
We'll need to add the following to this, so users know it was only added in 5.6:
```suggestion
     * Finds the closest matching format from a set of formats at the current selection.
     * <br>
     * <em>Added in TinyMCE 5.6</em>
```

Reply:
I think moxiedoc has @version however not sure if site templates have that though.

Reply:
Interesting... I'll let Tyler know so we can look into it :+1: